src/main/java/gregtech/common/tools/ToolHardHammer.java

```diff
@@ -88,10 +89,8 @@ public void convertBlockDrops(World world, BlockPos blockPos, IBlockState blockS
         ToolUtility.applyHammerDrops(world.rand, blockState, dropList, EnchantmentHelper.getEnchantmentLevel(Enchantments.FORTUNE, toolStack), player);
     }
 
-    @Override
-    public void addInformation(ItemStack stack, List<String> lines, boolean isAdvanced) {
-        lines.add(I18n.format("metaitem.tool.tooltip.hammer.extra_drop"));
```

Comment:
Why remove this tooltip instead of just rewording it to be `Drops Crushed materials` or something like that?